app/routes/search.py

```diff
@@ -0,0 +1,89 @@
+from typing import Union, Annotated
+from fastapi import HTTPException, Response, APIRouter, Query
+from ..dependencies.apimodels import RegistrationObject
+from sqlmodel import select
+from ..db_tools.models import User
+from ..db_tools.database import SessionDep
+from requests import post, get
+import base64
+import json
+import getpass
+import re
+import hashlib
+import os
+
+
+router = APIRouter()
+
+CLIENT_ID = os.getenv("SPOTIFY_ID")
+CLIENT_SECRET = os.getenv("SPOTIFY_SECRET")
+
```

Comment:
These should be declared in dependencies/constants.py and imported into the file.